doc/code/qml_fermi.rst

```diff
@@ -0,0 +1,89 @@
+qml.fermi
+=========
+
+Overview
+--------
+
+This module contains functions and classes for creating and manipulating fermionic operators.
+
+.. currentmodule:: pennylane.fermi
+
+.. automodapi:: pennylane.fermi
+    :no-heading:
+    :no-main-docstr:
+    :no-inherited-members:
+
+FermiC and FermiA
+-----------------
+
+The fermionic creation and annihilation operators are constructed with :class:`~pennylane.FermiC`
+and :class:`~pennylane.FermiA`, respectively, by passing the index of the orbital that the fermionic
+operator acts on. For instance, the operators :math:`a^{\dagger}_0` and :math:`a_3` are constructed as
+
+>>> qml.FermiC(0)
+a⁺(0)
+>>> qml.FermiA(3)
+a(3)
+
+These operators can be multiplied by each other to create :math:`n`-orbital fermionic operators such
+as :math:`a^{\dagger}_0 a_0 a^{\dagger}_3 a_3` that we call a Fermi word.
+
+>>> qml.FermiC(0) * qml.FermiA(0) * qml.FermiC(3) * qml.FermiA(3)
+a⁺(0) a(0) a⁺(3) a(3)
+
+The Fermi words can be linearly combined to create fermionic operators that we call a Fermi
+sentence. For instance, a fermionic Hamiltonian such as
+:math:`H = 1.2 a^{\dagger}_0 a_0  + 2.3 a^{\dagger}_3 a_3` can be constructed as
+
+>>> h = 1.2 * qml.FermiC(0) * qml.FermiA(0) + 2.3 * qml.FermiC(3) * qml.FermiA(3)
+>>> h
+1.2 * a⁺(0) a(0)
++ 2.3 * a⁺(3) a(3)
+
+Mapping to qubit operators
+--------------------------
+
+The fermionic operators can be mapped to the qubit basis by using the
+:func:`~pennylane.jordan_wigner` function. This function can be used to map creation and
+annihilation operators as well as Fermi words and Fermi sentences.
+
+>>> qml.jordan_wigner(qml.FermiA(1))
+(0.5*(PauliZ(wires=[0]) @ PauliX(wires=[1])))
++ (0.5j*(PauliZ(wires=[0]) @ PauliY(wires=[1])))
+
+>>> qml.jordan_wigner(qml.FermiC(1) * qml.FermiA(1))
+((0.5+0j)*(Identity(wires=[1])))
++ ((-0.5+0j)*(PauliZ(wires=[1])))
+
+>>> f = 0.5 * qml.FermiC(1) * qml.FermiA(1) + 0.75 * qml.FermiC(2) * qml.FermiA(2)
+>>> qml.jordan_wigner(f)
+((0.625+0j)*(Identity(wires=[1])))
++ ((-0.25+0j)*(PauliZ(wires=[1])))
++ ((-0.375+0j)*(PauliZ(wires=[2])))
+
+FermiWord and FermiSentence
+---------------------------
+
+Fermi words and Fermi sentences can also be constructed directly with
+:class:`~pennylane.fermi.FermiWord` and :class:`~pennylane.fermi.FermiSentence` by passing
+dictionaries that define the fermionic operators.
+
+For Fermi words, the dictionary items define the fermionic creation and annihilation operators.
+Each creation or annihilation operator is defined by a tuple of two integers representing its
+position in the Fermi word and the orbital it acts on, and a symbol identifying its type. We use
+``'+'`` and ``'-'`` to denote creation and annihilation operators, respectively. The operator
+:math:`a^{\dagger}_0 a_0 a^{\dagger}_3 a_3` can then be constructed with
```

Comment:
So the keys of the dictionary are the tuples and the values are a tag determining whether the operator is an annihilation or creator operator. This could be expressed more directly here.

Reply:
Rephrased.